content/realtime/connection.textile

```diff
@@ -69,7 +69,7 @@ ably.connection.on('connected', new ConnectionStateListener() {
 });
 
 bc[csharp]. AblyRealtime ably = new AblyRealtime("{{API_KEY}}");
-ably.Connection.On(ConnectionState.Connected, args => {
+ably.Connection.On(ConnectionEvent.Connected, args => {
```

Comment:
I'm not a .NET expert, but looking at the repo [readme](https://github.com/ably/ably-dotnet#connection), the examples there seem to use `ConnectionState` and not `ConnectionEvent`. If so, the ably-dotnet readme also needs updating.  

Also, this code seems to be in `ConnectionStateListener()`. Should it be `ConnectioEventListener`? 

Just noticed, corresponding [reference guide](https://ably.com/documentation/realtime/connection#connection-state) looks incorrect:  It seems to have both `ConnectionState` and `ConnectionEvent`, but the documentation for `ConnectionEvent` just seems to duplicate the previous section for `ConnectionState`.

![image](https://user-images.githubusercontent.com/25511700/115230457-4a1c9a00-a10c-11eb-8868-ca5e31de932e.png)

@tomkirbygreen - would you be able to look into these items for .NET and confirm the situation please?